packages/react-portal/src/components/Portal/renderPortal.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import { PortalState } from './Portal.types';
+import { getSlots } from '@fluentui/react-utilities';
+
+/**
+ * Render the final JSX of Portal
+ */
+export const renderPortal = (state: PortalState) => {
+  const { slots, slotProps } = getSlots(state);
+
+  const content = <slots.root {...slotProps.root}>{state.children}</slots.root>;
```

Comment:
A fact that we render an additional element is a bit confusing for me. Do we have a requirement to do this?

---

In Northstar & [Material UI](https://github.com/mui-org/material-ui/blob/next/packages/material-ui-unstyled/src/Portal/Portal.js) such element is not rendered. In current case, it will create unexpected markup:

```tsx
<Portal as="footer"> // renders "footer"
  <PopupContent /> renders another "div"
</Portal>
```

It immediately raises a question: where classes and DOM attributes should be applied? To `Portal`?